packages/lib-classifier/src/components/Classifier/components/Layout/components/CenteredLayout/CenteredLayout.js

```diff
@@ -21,40 +21,35 @@ const Relative = styled(Box)`
 const ContainerBox = styled(Box)`
   display: flex;
   flex-direction: row;
-  gap: 1.25rem;
+  gap: 20px;
   justify-content: center;
 
   ${props => props.hasSurveyTask && css`
     @media screen and (min-width: 769px) and (max-width: 70rem) {
       flex-direction: column;
-      margin: 0;
     }
   `}
 
   // small screens
   @media screen and (max-width: 768px) {
     flex-direction: column;
-    margin: 0;
   }
 `
 
 const ViewBox = styled(Box)`
-  display: flex;
   flex-direction: row;
-  margin: 0;
+`
 
-  // small screens
-  @media screen and (max-width: 768px) {
-    margin: auto;
-  }
+const StyledImageToolbarContainer = styled.div`
+  min-width: 3rem;
+  width: 3rem;
 `
```

Comment:
Similarly, the StyledImageToolbarContainer shared in the other layouts is a styled `div`, not a styled Grommet Box.